content/ai/concepts/generative-ai/concepts/images/images.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Images'
+Description: GenAI can generate new images from existing text prompts and images. Because of a random seed, the images generated are unique creations. GenAI uses diffusion models to create these new unique images.
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
+  - 'Transformers'
+Tags:
+  - 'AI'
+CatalogContent:
+  - 'paths/machine-learning-engineer'
+  - 'intro-to-chatgpt'
+---
+
+**Generative images** is the generation of new unique images from text prompts and existing images. Because of a random seed, the images generated are unique creations. GenAI uses diffusion models to create these new unique images.
+
+## Diffusion Models
+
+Image creation is mainly based on a relatively new development called diffusion. What is a diffusion model?
+
+Until recently, Generative Adversarial Networks (GANs) were the standard way to create images. With this approach, random noise is used to create an image. The image is then fed into another network which is trained to determine the quality. With more iterations, both networks (adversaries) can improve.
+
+Diffusion models are instead inspired by the world of physics. Imagine a pot of coffee brewing in a large room. Eventually the molecules we can smell diffuse into the air and travel throughout. If later someone asked for another pot of coffee we could trace back the motion track of the molecules and create another pot.
+
+GenAI image platforms are trained on vast amounts of data. By using diffusion models, these platforms can turn natural language descriptions into photo-realistic images. Some of these platforms include:
+
+- DALL-E
+- Midjourney
```

Comment:
Midjourney was fun! Just have to be really specific with what you want 😆 

Great examples 👍